blazor/carousel/styles-and-appearance.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Style and Appearance in Blazor Carousel Component | Syncfusion
+description: Checkout and learn here all about style and appearance in Syncfusion Blazor Carousel component and more.
+platform: Blazor
+control: Carousel
+documentation: ug
+---
+
+# CSS Structure in Blazor Carousel Component
+
+The following content provides the exact CSS structure that can be used to modify the control’s appearance based on user preference.
+
+## Customizing partial slides size
+
+You can customise the partial slides area size by overriding the `e-carousel-item` class.
```

Comment:
You can customize the partial slide size by overriding the `e-carousel-item` CSS class.